src/main/java/com/project/stockexchangeappbackend/rest/UsersController.java

```diff
@@ -196,4 +199,16 @@ public String getUsers() {
                 .map(transaction -> mapper.map(transaction, TransactionDTO.class));
     }
 
+    @GetMapping("/{id}/order")
+    @PreAuthorize("hasRole('ADMIN')")
+    @ApiOperation(value = "List user's orders", response = StockDTO.class,
```

Comment:
widzę to, czekam na review i poprawię

Reply:
ok

Reply:
By the way, How did you know changes will be required?